models.js

```diff
@@ -10,3 +10,14 @@ exports.selectCategories = () => {
       return res.rows;
     });
 };
+
+exports.findReviewByID = (id) => {
+  const arr = [parseInt(id.review_id)];
```

Comment:
Your query to the database is a string, this may not need to be here as psql will handle whether it is a digit or not.